Types failing in `4.14.2`

Hi!

I updated from `4.13.1` to `4.14.2`, and now when we run `tsc --noEmit`, the following error is displayed

```
./node_modules/@segment/snippet/types.d.ts:12:13 - error TS2411: Property 'All' of type 'boolean | undefined' is not assignable to string index type 'boolean'.

12             All?: boolean
               ~~~
```

We are using typescript `3.8.3` .. a bit dated, but perhaps there is a mistake in this type definition?

--- a/types.d.ts
+++ b/types.d.ts
@@ -9,7 +9,6 @@ declare module '@segment/snippet' {
 
     interface LoadOptions {
         integrations?: {
-            All?: boolean
             [key: string]: boolean
         }
     }

